helm_deploy/values-prod.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 4
 
 image:
-  repository: quay.io/hmpps/prisonstaffhub
+  repository: quay.io/hmpps/digital-prison-services
   tag: latest
```

Comment:
I have not changed the name of the cert_secret2 property in this file
cert_secret2: prison-staff-hub-cert
As I assume that this is pulled from somewhere with this name

Reply:
see other comment above.